.github/workflows/mac.yml

```diff
@@ -45,12 +45,10 @@ jobs:
         uses: actions/setup-python@v1
```

Comment:
Do we really need to set up Python again?

Reply:
I think we need to install a python 3 version so that we have pip3 available.

Reply:
What surprises me is that it's done twice (if I'm not overlooking anything).

Reply:
Once for the first job, once for the second job. Jobs are run independently of each other.

Reply:
That makes sense. I missed that these are two separate jobs.